sm_value_store/include/sm/value_store/ValueStore.hpp

```diff
@@ -240,24 +238,26 @@ class ExtendibleValueStore : public ValueStore {
 
 class ExtendibleValueStoreRef : public ValueStoreRef {
  public:
-  ExtendibleValueStoreRef(sm::PropertyTree bpt);
+  ExtendibleValueStoreRef() : ValueStoreRef() {}
```

Comment:
Why no longer accept a property tree?

Reply:
It just moved a bit down, in consistency with ValueStoreRef (line 245)

Reply:
Ah, great! Then my merging wasn't prematurely after all :)!